docs/rotation_broadening/rotation_broadening.ipynb

```diff
@@ -27,7 +27,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    plt.ylabel(r"Flux density [erg/s/cm$^2$/$\AA$]")
</pre><p>Would be nicer with <code>^{-1}</code> notation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>